langServer/tstCompletion.ts

```diff
@@ -70,21 +174,23 @@ export async function getTstCompletionData(
       returnData.choiceList = scriptFeatureList;
     } else if (trigger == "COLON" && upperCaseLine == "TEST.SUBPROGRAM:") {
       // find closest TEST.UNIT above this line ...
-      const unitName = getNearest(
+      const unit = getNearest(
         currentDocument,
         "UNIT",
         completionData.position.line
       );
+
       // TBD will need to change how this is done during the fix for issue #170
```

Comment:
we can remove this comment now :)

Reply:
Removed in 265346f47f3d1a2bda59ea03cf5c5e455c889ea0.